src/helpers/getNewTodoId.ts

```diff
@@ -0,0 +1,7 @@
+import { Todo } from '../Types';
+
+export const getId = (todos: Todo[]) => {
```

Comment:
Use more descriptive name
```suggestion
export const getTodoId = (todos: Todo[]) => {
```